Automattic/sensei
# Fix not enrolled notice style in lm

Resolves https://github.com/Automattic/sensei/issues/7241

## Proposed Changes
* Updated styles of the notice that's showed on lessons when user is not enrolled to match the other notices.

## Testing Instructions


We don't have and exact design for this notice. So we've fixed it taking inspirations from other notices.

1. Create a course with a lesson
2. Try to access the lesson as a student without taking the course
3. Check the notice both from Course theme and other themes
4. Check all variations of course theme
5. Check in mobile view

#### Desktop
![Screenshot 2023-11-03 at 10 44 23 PM](https://github.com/Automattic/sensei/assets/6820724/80e2289b-ecba-4984-83a3-83a80f603b2d)

#### Mobile
![Screenshot 2023-11-03 at 10 44 36 PM](https://github.com/Automattic/sensei/assets/6820724/f27936f8-562f-436c-8bab-5efa1ff44e04)

## Pre-Merge Checklist
